src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+export const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state) => {
+      state.value -= 1;
+    },
+    removeBook: (state, action) => {
+      state.value += action.payload;
+    },
```

Comment:
- [ ] You're trying to update a value property in your `addBook` reducer, but there is no `value` property in your initial state. You're increasing `state.value`, which might not be the intended behavior. If you want to remove a book, you should filter it out from the `state.books` array.


```suggestion
 removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
```